zer0-os/zOS
# Create Invite Dialog

### What does this do?

Adds the invitation dialog for inviting your friends to use Messenger.

Note: hard codes a fake invitation code until the api endpoint is deployed.

### How do I test this?

Set the feature flag: `window.FEATURE_FLAGS.allowInvites = true` and reload the app. Open the messenger sidekick and you should be able to click the "Invite Friend" button at the bottom. Verify the invititation text is copied to your clipboard.
